metadata.json

```diff
@@ -98,12 +98,13 @@
         "12.04",
         "14.04",
         "16.04",
-        "18.04"
+        "18.04",
+        "20.04"
       ]
     }
   ],
   "dependencies": [
     {"name":"trlinkin/nsswitch","version_requirement":">= 2.1.0 < 3.0.0"},
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.22.0 < 6.0.0"}
```

Comment:
I see why we want to allow for stdlib6, though curious why the bumb from 4.22.0 to 4.25.0 as the minimum?

Reply:
I took it from #221. In one of your comments (https://github.com/ghoneycutt/puppet-module-pam/pull/221#discussion_r436889223) you said:
> Please change to ">= 4.25.0 < 7.0.0"